### ISSUE TITLE
could not train a well-performed model...

### ISSUE THREAD

**Issue by gaizixuan0128** [Lotayou/densebody_pytorch#43]
I have used the original code and the provided toy dataset and trained with enough time, but the predicted UV map I get seems wrong...
Does anyone encounter this problem? How to solve it? Thanks!

This is the UV map generated during the training process...(I have trained like, about 200 epochs...)
![202_01999](https://user-images.githubusercontent.com/24839231/62854537-30656580-bd22-11e9-9cd5-470491ce442c.png)

**Comment by Lotayou** [comment]
Actually this is not bad. Maybe you should tweak the learning rate a little bit and see if that helps. Also, you can use some of the visualizing methods within `UV_Map_Generator` to resample the mesh from predicted UV coordinate maps.

**Comment by Lotayou** [comment]
A pre-trained model has been released, please checkout the latest update.
